packages/beacon-node/src/chain/prepareNextSlot.ts

```diff
@@ -228,6 +235,20 @@ export class PrepareNextSlotScheduler {
 
         precomputeEpochTransitionTimer?.();
       }
+
```

Comment:
By placing the fcu call after epoch transition flow, I am curious if there is any occurrence where there will be less than 500ms left in the slot after the epoch transition flow is finished.

Reply:
since we offload shuffling computation to native in #7120, I haven't seen any case we only have 500ms left before the next epoch, the batch hash work will even make it better. This is holesky 1k node:

<img width="1331" alt="Screenshot 2025-02-10 at 15 58 48" src="https://github.com/user-attachments/assets/91af257f-43d9-44d2-9119-8b887e1ce658" />



in the worse case if that happen, this `fcu` call will happen right after epoch transition, before the next block comes so there should be no issue to EL